Lib/ufo2ft/postProcessor.py

```diff
@@ -357,6 +361,177 @@ def _subroutinize_with_cffsubr(cls, otf, cffVersion):
 
         return cffsubr.subroutinize(otf, cff_version=cffVersion, keep_glyph_names=False)
 
+    def apply_fontinfo(self):
+        """Apply the fontinfo data from the DesignSpace variable-font's lib to
+        the compiled font."""
+        import copy
+
+        logger.info("Applying variable-font info from DesignSpace lib")
+
+        # Create a temporary UFO and sets its fontinfo to the union of the main
+        # UFO's fontinfo and the DesignSpace variable-font’s info.
+        temp_ufo = type(self.ufo)()
+        temp_ufo.info = copy.copy(self.ufo.info)
+        for k, v in self.info.items():
+            setattr(temp_ufo.info, k, v)
+
+        # Build a temporary font with the only tables that can be modified with
+        # fontinfo, then merge relevant table attributes it into the main font.
+        compiler = InfoCompiler(temp_ufo, self.otf)
+        compiler.compile()
+
+
+class InfoCompiler(BaseOutlineCompiler):
+    info_tables = frozenset(
+        [
+            "head",
+            "hhea",
+            "name",
+            "OS/2",
+            "post",
+            "vhea",
+            "gasp",
+        ]
+    )
+
+    def __init__(self, ufo, otf):
+        self.orig_otf = otf
+        tables = self.info_tables & set(otf.keys())
+        super().__init__(ufo, tables=tables, glyphSet={}, glyphOrder=[])
+        if "gasp" in tables:
+            self.setupTable_gasp()
+
+    @staticmethod
+    def makeMissingRequiredGlyphs(*args, **kwargs):
+        return
+
+    def makeFontBoundingBox(self):
+        from ufo2ft.outlineCompiler import EMPTY_BOUNDING_BOX
+
+        return EMPTY_BOUNDING_BOX
+
+    def _set_attrs(self, tag, attrs):
+        temp = self.otf[tag]
+        orig = self.orig_otf[tag]
+        for attr in attrs:
+            if value := getattr(temp, attr, None):
```

Comment:
if the value == 0, the setattr will be skipped, no? I think you meant `is not None`